zeebe/zb-db/src/main/java/io/camunda/zeebe/db/impl/rocksdb/SharedResourcesHelper.java

```diff
@@ -12,25 +12,23 @@
 import org.rocksdb.RocksDB;
 import org.rocksdb.WriteBufferManager;
 
-public class SharedResourcesTestHelper implements AutoCloseable {
+public class SharedResourcesHelper implements AutoCloseable {
+  // used for tests that don't need a specific cache size.
   public static final long DEFAULT_CACHE_SIZE = 32L * 1024 * 1024;
-  public static final long DEFAULT_WRITE_BUFFER_SIZE = DEFAULT_CACHE_SIZE / 4;
+  private static SharedRocksDbResources sharedRocksDbResources;
```

Comment:
❓ does this need to be a class? it wraps the `SharedRocksDbResources`, but they are themselves an `AutoCloseable` so there' no need now probably